src/node_crypto.cc

```diff
@@ -661,16 +641,19 @@ void SecureContext::AddCACert(const FunctionCallbackInfo<Value>& args) {
     newCAStore = true;
   }
 
-  X509* x509 = LoadX509(env, args[0]);
-  if (!x509)
-    return;
-
-  X509_STORE_add_cert(sc->ca_store_, x509);
-  SSL_CTX_add_client_CA(sc->ctx_, x509);
-
-  X509_free(x509);
+  unsigned cert_count = 0;
+  if (BIO* bio = LoadBIO(env, args[0])) {
+    while (X509* x509 =  // NOLINT(whitespace/if-one-line)
```

Comment:
Is the following not possible?
```cpp
  if (BIO* bio = LoadBIO(env, args[0])) {
    X509* x509;
    while (x509 = PEM_read_bio_X509(bio, nullptr, CryptoPemCallback, nullptr)) {
     // ...
```
If not, I'd say how it is now isn't the prettiest but personally find it easier to logically follow. Which wins for me.